datahub-web-react/src/app/shared/TagStyleEntity.tsx

```diff
@@ -184,8 +186,24 @@ export default function TagStyleEntity({ urn, useGetSearchResults = useWrappedSe
     const { error, data, refetch } = useGetTagQuery({ variables: { urn } });
     const [updateDescription] = useUpdateDescriptionMutation();
     const [setTagColorMutation] = useSetTagColorMutation();
-    const entityAndSchemaQuery = `tags:"${data?.tag?.name}" OR fieldTags:"${data?.tag?.name}" OR editedFieldTags:"${data?.tag?.name}"`;
-    const entityQuery = `tags:"${data?.tag?.name}"`;
+    const entityUrn = data?.tag?.urn;
+    const entityFilters =
+        (entityUrn && [
+            {
+                field: 'tags',
+                values: [entityUrn],
+            },
+        ]) ||
+        [];
+    const entityAndSchemaFilters =
+        (entityUrn && [
+            ...entityFilters,
+            {
+                field: 'fieldTags',
+                values: [entityUrn],
+            },
+        ]) ||
+        [];
```

Comment:
do we need `editedFieldTags` here as well?

Reply:
We do not! Gabe wrote some logic on backend to expand

Reply:
nice!

Reply:
🪄